shared/desktop/renderer/remote-component-loader.js

```diff
@@ -177,7 +175,7 @@ const styles = {
     backgroundColor: globalColors.white,
   },
   loading: {
-    backgroundColor: globalColors.grey5,
+    backgroundColor: globalColors.grey,
```

Comment:
there never was a grey5! this is cause eslint / flow is working better everywhere